src/dataset.py

```diff
@@ -61,43 +76,61 @@ def __getitem__(self, i):
             img = np.concatenate((img, border), axis=0)
             img = torch.from_numpy(img)
 
-        if self.use_snow_i:
-            snow_index = utils.get_snow_index(img)
-            snow_index = np.expand_dims(snow_index, axis=0)
-            img = np.concatenate((img, snow_index), axis=0)
-            img = torch.from_numpy(img)
-
         if self.img_transform is not None:
             img = self.img_transform(img)
 
-        # default is 'glaciers' for original labels 
+        # default is 'glaciers' for origina/labels 
```

Comment:
I think there was a find and replace issue here or something.

Reply:
Thanks!